moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -416,6 +423,9 @@ class PlanningSceneMonitor : private boost::noncopyable
   planning_scene::PlanningSceneConstPtr scene_const_;
   planning_scene::PlanningScenePtr      parent_scene_; /// if diffs are monitored, this is the pointer to the parent scene
   boost::shared_mutex                   scene_update_mutex_; /// mutex for stored scene
+  ros::Time                             last_update_time_; /// Last time the state was updated
```

Comment:
That's exactly what is confusing - the naming is not unique because the comment indicates it deals with *state* updates, not scene updates as one might assume from the class name PlanningSceneMonitor